superset-frontend/package.json

```diff
@@ -51,8 +51,8 @@
     "dev": "webpack --mode=development --color --watch",
     "dev-server": "cross-env NODE_ENV=development BABEL_ENV=development node --max_old_space_size=4096 ./node_modules/webpack-dev-server/bin/webpack-dev-server.js --mode=development",
     "format": "npm run _prettier -- --write",
-    "lint": "npm run _lint && npm run type",
-    "lint-fix": "npm run _lint -- --fix && npm run type",
+    "lint": "npm run eslint -- . && npm run type",
```

Comment:
maintaining backward compatibility, though I think we should deprecate eventually as it's unclear that you're running two commands here, and arguably `tsc` isn't really linting.